go/vt/binlog/binlogplayer/binlog_player.go

```diff
@@ -733,7 +733,9 @@ func MysqlUncompress(input string) []byte {
 		return nil
 	}
 	var outputBytes bytes.Buffer
-	io.Copy(&outputBytes, reader)
+	if _, err := io.Copy(&outputBytes, reader); err != nil {
```

Comment:
@rohit-nayak-ps I returned `nil` instead of error here following along the `outputBytes.Len()` check immediately below returning `nil` instead of error. Let me know if you think this should return an error instead.

Reply:
that is fine